tests/unit_test/channels/whatsapp_test.py

```
@@ -0,0 +1,270 @@
+from mock import patch
+import pytest
+import os
+
+from kairon.shared.utils import Utility
+from mongoengine import connect
+
+class TestWhatsappHandler:
+
+    @pytest.fixture(autouse=True, scope='class')
+    def setup(self):
+        os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)